readme.md

```diff
@@ -252,22 +225,26 @@ generally not a good idea to change the prototype directly by assigning to the
 **Note**: `.prototype` is a property on constructor functions, while `.__proto__` is the
 property on objects, and is used in the lookup chain.
 
-### Exercise: Car Constructor with Prototypes (10 minutes - 90/150)
+### You-Do Exercise: Car Constructor with Prototypes (10 min)
 
 Update the constructor function for our car to define the methods on prototypes
 rather than on the individual instances themselves.
 
 See `car.js` for a solution.
 
-## Exercise: Monkey (20 minutes - 100/150)
+## Break (10 min)
+
+### You-Do Exercise: Monkey (20 min)
 
-Work on the [OOP Monkey Exercise](https://github.com/ga-dc/oop_monkey/tree/js) (same as in Ruby, now in JS!).
+Work on the [OOP Monkey Exercise](https://github.com/ga-dc/oop_monkey/tree/js)
 
 Make sure you check out the `js` branch before beginning!
 
 `$ git checkout js`
 
-## Inheritance (15 minutes - 120/150)
+**Please Note:** You will be using a testing framework called Jasmine in this exercise that you will learn more about later on in the course. For now, we will only be creating functions in order to get the test to pass
+
```

Comment:
Ignore my earlier comment about testing! Although I think this will also be the first time they have been exposed to `npm`, which also might need some explaining.